01_01_Basic_Rules.txt

```diff
@@ -555,13 +555,15 @@ negative effects.
 
 - Will: defend against a mind affect (Ex: throw off hopelessness after crashing in an escape pod)
 - Shock: maintain composure after serious damage (Ex: stay conscious after being shot by a high caliber round)
-- Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another).
+- Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another)
+- Awareness: passive knowledge and understanding of surroundings (Ex: realizing someone has walked up behind you)

```

Comment:
I just don't like using the word passive here, in player documentation, for some reason. Comprehension of ones surroundings does require some active and critical thinking / processing, even if its no enough for us to represent it mechanically. Probably not a merge blocker.

Reply:
I think the description will work just as well without it there.